src/scene/vcModel.cpp

```diff
@@ -312,7 +312,14 @@ void vcModel::HandleImGui(vcState *pProgramState, size_t * /*pItemID*/)
   if (repackMatrix)
   {
     m_sceneMatrix = udDouble4x4::translation(m_pivot) * udDouble4x4::rotationQuat(orientation, position) * udDouble4x4::scaleUniform(scale.x) * udDouble4x4::translation(-m_pivot);
-    vcProject_UpdateNodeGeometryFromCartesian(&pProgramState->activeProject, m_pNode, *m_pCurrentZone, vdkPGT_Point, &position, 1);
+
+    if (m_pCurrentZone != nullptr)
+      vcProject_UpdateNodeGeometryFromCartesian(&pProgramState->activeProject, m_pNode, *m_pCurrentZone, vdkPGT_Point, &position, 1);
+    else if (m_pPreferredProjection != nullptr)
+      vcProject_UpdateNodeGeometryFromCartesian(&pProgramState->activeProject, m_pNode, *m_pPreferredProjection, vdkPGT_Point, &position, 1);
```

Comment:
I think this will turn out to be not required as we should be in `pProgramState->geozone` at this point anyway